charts/proxmox-csi-plugin/templates/namespace.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if ne .Release.Namespace "kube-system" }}
+{{- if and .Values.createNamespace (ne .Release.Namespace "kube-system") }}
```

Comment:
I don't undestand the meaning of the `createNamespace` boolean. In my [#95 issue](https://github.com/sergelogvinov/proxmox-csi-plugin/issues/95), I've demonstrated that the creation of the namespace doesn't work.
Is the boolean aim to disable the namespace creation only ?
Also, if we don't enter in th if condition, you don't indicate int the README to add the security labels to the namespace. Are they mandatory ?

Reply:
Yeah, I was too focused on the review. I said in my issue that you just want to disable the namespace creation.

My bad. :-)

Reply:
You are correct; we need to clarify the required label if someone wants to create namespace manually.